pymc3/step_methods/nuts.py

```diff
@@ -0,0 +1,238 @@
+from .quadpotential import quad_potential
```

Comment:
Just delete it? I was playing with rebasing for half an hour and ended up with `rm -rf pymc3; git clone ...`